app/assets/javascripts/actions/story.js

```diff
@@ -88,6 +88,21 @@ export const setLoadingStory = (id, from) => ({
   from
 });
 
+export const confirmBeforeSaveIfNeeded = async (story, confirm, needConfirmation, callback) => {
+  const confirmStoryChange = story => confirm(I18n.t('story.definitive_sure', { action: 'change' }));
```

Comment:
This approach won't work for other languages :sweat_smile: 

![Screenshot from 2019-11-13 14-21-08](https://user-images.githubusercontent.com/4325587/68787746-dd50c680-0620-11ea-9fce-9bf5962d4845.png)

Reply:
Also, can we fix the message here https://github.com/Codeminer42/cm42-central/blob/master/config/locales/pt-BR.yml#L124? It's as "Essa ações", it should be "Essa ação" :smile:

Reply:
Oh, sure!